From 75c06e3e2a4760b36fffd95cdf5535b8ad73c481 Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Mon, 25 Feb 2019 15:27:54 +0800
Subject: [PATCH 20/24] missing_type.h: add __compar_d_fn_t definition

Fix the following compile failure:
src/basic/util.h:71:18: error: unknown type name '__compar_d_fn_t'; did you mean '__compar_fn_t'?

Upstream-Status: Inappropriate [musl specific]

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 src/basic/missing_type.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/basic/missing_type.h b/src/basic/missing_type.h
index 7d7c1e4..85902ab 100644
--- a/src/basic/missing_type.h
+++ b/src/basic/missing_type.h
@@ -13,6 +13,7 @@
 
 #ifndef __GLIBC__
 typedef int (*comparison_fn_t)(const void *, const void *);
+typedef int (*__compar_d_fn_t) (const void *, const void *, void *);
 #endif
 
 #ifndef __COMPAR_FN_T
-- 
2.7.4

